core/src/main/java/org/elasticsearch/index/query/MissingQueryBuilder.java

```diff
@@ -60,16 +82,16 @@ public MissingQueryBuilder existence(boolean existence) {
         return this;
     }
 
+    public boolean existence() {
```

Comment:
Same here. Comment would help to know what it means if true/false is returned here.